CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.6)
+
+project (elfin)
+
+set(CMAKE_CXX_STANDARD 17)
+
+add_subdirectory(elf elfdir)
+add_subdirectory(dwarf dwarfdir)
+add_subdirectory(examples examplesdir)
```

Adding an immediate subdirectory in the source tree as a different directory in the build directory is not necessary.
```suggestion
add_subdirectory(elf)
add_subdirectory(dwarf)
add_subdirectory(examples)
```